--- gmime-1.90.0/Makefile.am.orig	Fri Jun 28 07:59:55 2002
+++ gmime-1.90.0/Makefile.am	Thu Jul 18 12:25:39 2002
@@ -222,19 +222,19 @@
 ## the value NONE in configure if the user doesn't specify them (this
 ## is an autoconf feature, not a bug).
 
-confexecdir = $(libdir)
-confexec_DATA = gmimeConf.sh
-
 CLEANFILES = gmimeConf.sh
 
 confexecdir = $(libdir)
 confexec_DATA = gmimeConf.sh
 
+aclocalm4dir = $(datadir)/aclocal
+aclocalm4_DATA = gmime.m4
+
 EXTRA_DIST = 			\
 	gmimeConf.sh.in 	\
 	gmime-2.0.pc.in		\
 	gmime.spec.in 		\
-	gmime.m4 		\
+	$(aclocalm4_DATA)	\
 	iconv-detect.c		\
 	zentimer.h 		\
 	zenprofiler.h
@@ -262,6 +262,3 @@
 
 $(libgmime_2_0_la_SOURCES): $(srcdir)/libgmime
 
-install-data-local:
-	-mkdir -p $(DESTDIR)$(datadir)/aclocal
-	$(INSTALL_DATA) $(srcdir)/gmime.m4 $(DESTDIR)$(datadir)/aclocal/gmime.m4
