diff -urN gmime-2.1.5.orig/configure.in gmime-2.1.5/configure.in
--- gmime-2.1.5.orig/configure.in	2004-05-23 13:13:09.738451600 +0200
+++ gmime-2.1.5/configure.in	2004-05-23 13:14:19.489847776 +0200
@@ -120,7 +120,7 @@
 dnl *** Checks for libnsl     ***
 dnl *****************************
 LIBNSL=""
-AC_CHECK_LIB(nsl, gethostbyname, LIBNSL="-lnsl")
+AC_CHECK_FUNC(gethostbyname, ,[AC_CHECK_LIB(nsl, gethostbyname, LIBNSL="-lnsl")])
 
 dnl Check for libiconv
 AM_ICONV()
