--- gmime-2.6.23/configure.ac.orig	2022-04-25 21:54:09.221548521 +0200
+++ gmime-2.6.23/configure.ac	2022-04-25 21:55:36.214410573 +0200
@@ -550,13 +550,13 @@ dnl ****************************
 dnl *** Checks for libsocket ***
 dnl ****************************
 LIBSOCKET=""
-AC_CHECK_LIB(socket, getaddrinfo, LIBSOCKET="-lsocket")
+AC_CHECK_FUNC(getaddrinfo,,[AC_CHECK_LIB(socket, getaddrinfo, LIBSOCKET="-lsocket")])
 
 dnl *************************
 dnl *** Checks for libnsl ***
 dnl *************************
 LIBNSL=""
-AC_CHECK_LIB(nsl, getaddrinfo, LIBNSL="-lnsl")
+AC_CHECK_FUNC(getaddrinfo,,[AC_CHECK_LIB(nsl, getaddrinfo, LIBNSL="-lnsl")])
 
 dnl ******************************************
 dnl *** Checks for GpgME needed for S/MIME ***
