--- gmime-1.90.0/gmime-multipart-encrypted.c.orig	Wed Jun 26 06:43:17 2002
+++ gmime-1.90.0/gmime-multipart-encrypted.c	Thu Aug 29 15:28:13 2002
@@ -50,7 +50,7 @@
 static void multipart_encrypted_remove_header (GMimeObject *object, const char *header);
 static void multipart_encrypted_set_content_type (GMimeObject *object, GMimeContentType *content_type);
 static char *multipart_encrypted_get_headers (GMimeObject *object);
-static int multipart_encrypted_write_to_stream (GMimeObject *object, GMimeStream *stream);
+static ssize_t multipart_encrypted_write_to_stream (GMimeObject *object, GMimeStream *stream);
 
 
 static GMimeMultipartClass *parent_class = NULL;
--- gmime-1.90.0/gmime-multipart-signed.c.orig	Thu Jun 27 04:39:13 2002
+++ gmime-1.90.0/gmime-multipart-signed.c	Thu Aug 29 15:30:49 2002
@@ -51,7 +51,7 @@
 static void multipart_signed_remove_header (GMimeObject *object, const char *header);
 static void multipart_signed_set_content_type (GMimeObject *object, GMimeContentType *content_type);
 static char *multipart_signed_get_headers (GMimeObject *object);
-static int multipart_signed_write_to_stream (GMimeObject *object, GMimeStream *stream);
+static ssize_t multipart_signed_write_to_stream (GMimeObject *object, GMimeStream *stream);
 
 
 static GMimeMultipartClass *parent_class = NULL;
