--- gmime-2.1.10/configure.in.orig	2005-01-11 22:36:39.000000000 +0100
+++ gmime-2.1.10/configure.in	2005-01-22 17:24:02.701878520 +0100
@@ -184,13 +184,13 @@
 dnl *** Checks for libsocket ***
 dnl ****************************
 LIBSOCKET=""
-AC_CHECK_LIB(socket, getaddrinfo, LIBSOCKET="-lsocket -lnsl")
+AC_CHECK_FUNC(getaddrinfo,,[AC_CHECK_LIB(socket, getaddrinfo, LIBSOCKET="-lsocket -lnsl")])
 
 dnl *************************
 dnl *** Checks for libnsl ***
 dnl *************************
 LIBNSL=""
-AC_CHECK_LIB(nsl, getaddrinfo, LIBNSL="-lnsl")
+AC_CHECK_FUNC(getaddrinfo,,[AC_CHECK_LIB(nsl, getaddrinfo, LIBNSL="-lnsl")])
 
 dnl ****************************
 dnl *** Enable Mono bindings ***
